back/server/prisma/datamodel.prisma

```diff
@@ -65,6 +81,35 @@ type Stake {
   totalStake: String!
 }
 
+type Motion {
+  id: Int! @id(strategy: SEQUENCE) @sequence(name: "MOTION_SEQUENCE" initialValue:1 allocationSize:1)
+  author: String!
+  memberCount: Int!
+  metaDescription: String!
+  method: String!
+  motionProposalArguments: [MotionProposalArgument]! @relation(name: "ArgumentForMotionProposal")
+  motionProposalHash: String!
+  motionProposalId: Int! @unique
+  preimage: Preimage @relation(name: "PreimageForMotion")
+  preimageHash: String
+  section: String!
+  status: [MotionStatus]! @relation(name: "MotionStatusFromMotion")
```

Comment:
Hmm this datamodel was actually changed in #167 
That's the "conflicts" I was talking about. In the end it's not a conflict, just something that needs to be synced.